.circleci/config.yml

```diff
@@ -17,20 +17,18 @@ workflows:
           filters: *filters
       - orb-tools/review:
           filters: *filters
+          exclude: RC010
```

Comment:
Ignore snake_case requirements. This would be a breaking change for the Cypress app to introduce this check.